crypto/src/main/kotlin/net/corda/v5/crypto/SecureHash.kt

```diff
@@ -36,6 +37,22 @@ class SecureHash(val algorithm: String, bytes: ByteArray) : OpaqueBytes(bytes) {
                 SecureHash(algorithm, data)
             }
         }
+        fun secureHash(bytes: ByteArray, algorithm: String = "SHA-256"): SecureHash {
+            val messageDigest = MessageDigest.getInstance(algorithm) // CORE-4984 review: caching messageDigest?
```

Comment:
This is unfortunately not correct way to get hashers. We have a pluggable architecture for that and you need to talk to @alexey-kadyrov-r3 on correct way to get the algorithm

Reply:
You need to use net.corda.v5.crypto.DigestService which is OSGi component. Its methods return the SecureHash. Also the double hash is the algorithm now with the name SHA-256D